README.md

```diff
@@ -24,6 +24,7 @@ export AUTH_FILE=<path to service-account.json>
 export GCS_BUCKET=prow
 export GITHUB_ORG=prometheus
 export GITHUB_REPO=prometheus
+export GRAFANA_ADMIN_PASSWORD=password
```

Comment:
can we have this as  ```export GRAFANA_ADMIN_PASSWORD=$(openssl rand -hex 20)```

Reply:
If we set it to random value then the user deploying grafana will not know the admin password.
Will be unable to login to grafana-admin portal

Reply:
they can always do `echo $GRAFANA_ADMIN_PASSWORD`

Reply:
usually this will also be added to the Lastpass account.